### PR TITLE
Order Feature with each order having a group of images (Table)

### PR DESCRIPTION
**Pull request by SalmanMughal789**
This pull request introduces a exciting feature to the codebase, enhancing user interactivity by allowing users to add, and delete orders, each containing a separate table. The new functionality improves the user experience by providing a clear and organized way to manage order data.

   1.  Add Orders:
        1. Users can now add new orders, each associated with its own table.
        2. This improves overall user interactivity by providing a more organized view of orders.

  2.  Edit Order Labels:
       1. Users have the ability to edit the labels for each order.
        2. This allows for better customization and clarity of order information.

  3.  Delete Orders:
        1. Users can delete orders.
        2. Deleting an order also removes the corresponding table, ensuring data integrity.
        
    
        
image

Closed #36

### CODE REVIEW

File: frontend/components/invoices/Order.jsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        dataIndex: col.dataIndex,
+        title: col.title,
+        handleSave,
+      }),
+    };
+  });
+
+  return (
+    <>
+    <RemoveModal deleteRow={deleteRow} setDeleteRow={setDeleteRow} />
```
[the remark this answers is not in the document]
Comment (SalmanMughal789) [reply frontend/components/invoices/Order.jsx:177]: The order is a separate table having its own state for storing the deleteKey. This remove component is mounted in order as well as in parent. It does not lead to inconsistency as far as I have tested. We can move that separatThe order is a separate table having its own state for storing the deleteKey. This remove component is mounted in order as well as in parent. It does not lead to inconsistency as far as I have tested. We can move that separate component to parent but how would we then render it inside order table and manage it's state ? We would have to write duplicate code. Please guide me in it.

[the remark this answers is not in the document]
Comment (Waqar-SE) [reply frontend/components/invoices/Order.jsx:177]: Yes, let's have a call for it at 3:15 and I'll show you how we can do it.

[the remark this answers is not in the document]
Comment (SalmanMughal789) [reply frontend/components/invoices/Order.jsx:177]: Okay !

File: frontend/src/index.css [hunk 2]
```diff
@@ -15,6 +15,14 @@
     @apply bg-accent hover:!bg-hover-color-accent;
   }
 
+  /* Order Table */
```
[the remark this answers is not in the document]
Comment (SalmanMughal789) [reply frontend/src/index.css:18]: we can use tailwind but how can we target table classses like .ant-table-thead using tailwind as the table is rendered dynamically only the <Table /> is all we get

**Comment by SalmanMughal789** [comment]
> * Upon entering the height and width manually, the amount doesn't update automatically.
> 
>     * Disable pagination.
> 
>     * Make the invoice footer details more visible and easy to read. (Bold numbers and maybe give different colors.)

I have removed the pagination by setting pagination={false} 
height and width are updating amount and area automatically in real time
I have done some cosmetic improvements to address the visibility of footer of invoice

File: frontend/src/index.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (Waqar-SE) [reply frontend/src/index.css:19]: It was not even used.

**Pull request merged**